mscore/inspector/inspector_hairpin.ui

```diff
@@ -296,14 +296,20 @@
      <item row="2" column="0">
       <widget class="QCheckBox" name="hairpinSingleNote">
        <property name="text">
-        <string>Single note (de)crescendo</string>
+        <string>Play hairpin on single notes</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QToolButton" name="resetHairpinSingleNote">
+       <property name="toolTip">
+        <string>Reset value</string>
+       </property>
+       <property name="accessibleName">
+        <string>Reset playing hairpin on single notes</string>
```

Comment:
Good, almost... Please make it `Reset 'Play hairpin on single notes' value` to make it consistent with the others (a recent change)